### PR TITLE
Nutanix Prism Central UI cases automation support for virt-who plugin

### PR DESCRIPTION
**Pull request by yanpliu**
Hey,
Realize Nutanix Prism Central UI cases automation support for virt-who plugin.

Nutanix UI Cases:PASS
```
(robottelo_vv) [yanpliu@yanpliu robottelo]$ pytest ./tests/foreman/virtwho/ui/test_nutanix.py -k test_positive_prism_central_prism_central_option --disable-pytest-warnings -q

12 deselected, 4 warnings in 0.03s
2022-09-07 16:19:17 - robottelo.collection - INFO - Processing test items to add testimony token markers
(robottelo_vv) [yanpliu@yanpliu robottelo]$ pytest ./tests/foreman/virtwho/ui/test_nutanix.py -k test_positive_prism_central_prism_centr^Cq
(robottelo_vv) [yanpliu@yanpliu robottelo]$ pytest ./tests/foreman/virtwho/ui/test_nutanix.py -k test_positive_prism_central_deploy_configure_by_script --disable-pytest-warnings -q
.                                                                                                                                                                                                           [100%]
1 passed, 11 deselected, 9 warnings in 355.37s (0:05:55)
(robottelo_vv) [yanpliu@yanpliu robottelo]$ pytest ./tests/foreman/virtwho/ui/test_nutanix.py -k  test_positive_prism_central_deploy_configure_by_id --disable-pytest-warnings -q
.                                                                                                                                                                                                           [100%]
1 passed, 11 deselected, 9 warnings in 348.04s (0:05:48)

```

### CODE REVIEW

File: tests/foreman/virtwho/ui/test_nutanix.py [hunk 1]
```diff
@@ -150,3 +150,125 @@ def test_positive_hypervisor_id_option(self, default_org, session, form_data):
...
+        name = gen_string('alpha')
+        form_data['name'] = name
+        form_data['hypervisor_content.prism_flavor'] = "Prism Central"
+        with session:
+            session.virtwho_configure.create(form_data)
+            values = session.virtwho_configure.read(name)
+            script = values['deploy']['script']
+            hypervisor_name, guest_name = deploy_configure_by_script(
+                script, form_data['hypervisor_type'], debug=True, org=default_org.label
+            )
```
Comment (Griffin-Sullivan) [inline tests/foreman/virtwho/ui/test_nutanix.py:225]: It seems like this is the only difference between this test and the one above. Maybe you could add some helper functions to reduce the repeated lines?

Comment (yanpliu) [reply tests/foreman/virtwho/ui/test_nutanix.py:225]: > It seems like this is the only difference between this test and the one above. Maybe you could add some helper functions to reduce the repeated lines?
@Griffin-Sullivan I really total agree with you, I found there are some redundant code in test cases, not in nutanix test cases, also exist in other hypervisor, next I will submit another optimization PR, really thanks your good suggestion.

**Review by Griffin-Sullivan — APPROVED** [review]

File: tests/foreman/virtwho/ui/test_nutanix.py [hunk 2]
```diff
@@ -150,3 +150,125 @@ def test_positive_hypervisor_id_option(self, default_org, session, form_data):
                 assert get_configure_option('hypervisor_id', config_file) == value
             session.virtwho_configure.delete(name)
             assert not session.virtwho_configure.search(name)
+
```
Comment (shubhamsg199) [inline tests/foreman/virtwho/ui/test_nutanix.py:153]:
```suggestion
@pytest.mark.tier2
```

**Review by shubhamsg199 — APPROVED** [review]
Ack, one test marker you missed.

**Review by Gauravtalreja1 — APPROVED** [review]

**Comment by yanpliu** [comment]
Hey, Can somebody help to merge this PR? Thanks

**Comment by yanpliu** [comment]
@Gauravtalreja1  Could you please help to merge this PR, there are 3 person approved it, many thanks

**Review by Gauravtalreja1 — APPROVED** [review]
ACK, @yanpliu , This PR will need one more review from @SatelliteQE/robottelo-tier-2-reviewers  to get merged, Thanks :)

**Review by latran — APPROVED** [review]

**Pull request merged**